queries/indexing/properties-text-search.rq

```diff
@@ -0,0 +1,40 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX dc: <http://purl.org/dc/elements/1.1/>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+
+select
+?propertyURI 
+(group_concat(?skosDefinition; separator=", ") as ?skosDefinitions)
+(group_concat(?editorialNote; separator=", ") as ?editorialNotes)
+(group_concat(?example; separator=", ") as ?examples)
```

Comment:
I wonder if, from this set of editorial properties, some are also used in other datasets, try ?.